Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java

```diff
@@ -64,6 +70,9 @@
     @NotNull
     private ImmutableSortedSet<? extends IClusterDetail> clusters = ImmutableSortedSet.of();
 
+    @NotNull
+    private ImmutableSortedSet<? extends IClusterDetail> rawClusters = ImmutableSortedSet.of();
```

Comment:
Needn't to introduce the field `rawCluster`, just filter the list in `getClusters()`